play-zipkin-tracing/build.sbt

```diff
@@ -63,11 +63,12 @@ lazy val core = (project in file("core")).
     name := "play-zipkin-tracing-core",
     libraryDependencies ++= Seq(
       "commons-lang" % "commons-lang" % "2.6",
-      "io.zipkin.brave" % "brave" % "4.8.0",
-      "io.zipkin.reporter2" % "zipkin-sender-okhttp3" % "2.0.0",
-      "org.scalatest" %% "scalatest" % "3.0.3" % "test"
+      "io.zipkin.brave" % "brave" % "4.8.1",
+      "io.zipkin.reporter2" % "zipkin-sender-okhttp3" % "2.0.2",
+      "io.zipkin.brave" % "brave-tests" % "4.8.1" % "test"
     )
   )
+// TODO: make sbt run junit tests
```

Comment:
@takezoe can you help on this?

Reply:
@adriancole Sure. I will try to run Brave test.